tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -0,0 +1,205 @@
+assert = chai.assert
+
+class FakeHighlight
+
+  constructor: (@segment, @anchor, @page) ->
+
+  @createFrom: (segment, anchor, page) ->
+    new FakeHighlight segment, anchor, page
```

Comment:
> This would be simpler to just stub the `createFrom` method in a `beforeEach` block.

I did that in https://github.com/hypothesis/h/commit/34aa96825036caeca82da5fe59fcbe317c34e7be.